modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -192,7 +192,7 @@ export async function save(
 		const errorMessage =
 			(type && ERRORS[type]) ??
 			title ??
-			Liferay.Language.get('an-error-occurred');
```

Comment:
Hi @GabrielHenriquedev, I don't think this is the best way to solve the issue. This errorMessage can be displayed for any error that occur during the save process, not only when the key contain forbidden characters. It seems to me that the problem is related with the title, that should bring the correct message to be displayed. Maybe it's a back-end issue, something similar to this https://github.com/liferay-objects/liferay-portal/pull/935.